src/translations/en/authorization.json

```
@@ -0,0 +1,8 @@
+{
+  "back_to_home": "Back to home",
+  "forbidden": "Forbidden",
+  "forbidden_description": "You are not authorized to see the content on this page. Contact administrator if this is erroneous.",
```

hahah, meget fancy med `erroneous`. Kanskje vi ikke skal anta at brukere kan dette ordet?
```suggestion
  "forbidden_description": "You are not authorized to see the content on this page. Contact the administrator if you think this is wrong.",
```

---

haha, ble veldig glad i "erroneous" ila masterskriving! Thus, I still use it. Hence, I appear schmart.